docs/www/rpk-commands.md

```diff
@@ -161,6 +161,10 @@ Usage:
   rpk topic set-config <topic> <key> [<value>] [flags]
 ```
 
+## cluster
+
+### info
+
```

Comment:
Should we add some info about the command here?

Reply:
Oh, yeah. Good catch